python/py-tensorflow/Portfile

```diff
@@ -11,7 +11,7 @@ PortGroup           xcode_workaround               1.0
 
 name                py-tensorflow
 version             2.2.0
-revision            0
+revision            1
```

Comment:
Revision bumps appear to be necessary because the change in NumPy 1.19.x has been described as [ABI-breaking](https://github.com/tensorflow/tensorflow/issues/40688#issuecomment-647846011).

Reply:
Tensorflow ports have since been updated, although still do not have patch for NumPy 1.19.x and should fail to build; NumPy is no a reason to revbump.